### PR TITLE
Added browsing a sub directory

### LINKED ISSUE #7594: How do you get the directory listing of a subdirectory?
Great example, but it only shows how to get the directory listing of the root directory on the ftp server (or the default login directory). A great addition would be show how to issue a remote "cd" command before getting the directory listing.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f7e276f6-b096-bb7d-fc03-f4b09a10de43
* Version Independent ID: 3d15a865-75ab-e43d-0a95-399f3db7de5a
* Content: [How to: List directory contents with FTP](https://docs.microsoft.com/en-us/dotnet/framework/network-programming/how-to-list-directory-contents-with-ftp)
* Content Source: [docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md](https://github.com/dotnet/docs/blob/master/docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md)
* Product: **dotnet-framework**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

### PR DESCRIPTION
**Pull request by yashints**
Added code for browsing sub directory

## Summary

Added examples for browsing sub directory

Fixes #7594

### CODE REVIEW

File: docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md [hunk 1]
```diff
@@ -73,4 +73,14 @@ Namespace Examples.System.Net
         End Sub
     End Module
 End Namespace
-```
\ No newline at end of file
+```
+
+If you need to browse a specific directory, just add the directory to the end of URI you're using in `WebRequest.Create` method:
+
+````csharp
```
Comment (karelz) [inline docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md]: Extra ` breaks formatting

File: docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md [hunk 2]
```diff
@@ -73,4 +73,14 @@ Namespace Examples.System.Net
         End Sub
     End Module
 End Namespace
-```
\ No newline at end of file
+```
+
+If you need to browse a specific directory, just add the directory to the end of URI you're using in `WebRequest.Create` method:
```
Comment (karelz) [inline docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md]: browse -> list ... to keep the vocabulary consistent

**Review by karelz — CHANGES_REQUESTED** [review]
Thanks for improving the sample!

**Review by karelz — APPROVED** [review]

File: docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md [hunk 3]
```diff
@@ -73,4 +73,14 @@ Namespace Examples.System.Net
         End Sub
     End Module
 End Namespace
-```
\ No newline at end of file
+```
+
+If you need to list a specific directory, just add the directory to the end of URI you're using in `WebRequest.Create` method:
```
Comment (mairaw) [inline docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md]: nit: end of URI in -> end of **the** URI in **the**

**Review by mairaw — CHANGES_REQUESTED** [review]
Found a small nit to fix before we merge! Thanks @yashints!

File: docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md (hunk 3 above)
Comment (mairaw) [reply docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md]: One more small thing.
`WebRequest.Create` -> replace with the following syntax to get a link to the API docs:
`<xref:System.Net.WebRequest.Create%2A>`

Comment (mairaw) [reply docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md]: As is. It’s to represent a *. You can find more info at 
https://github.com/dotnet/docs/blob/master/styleguide/template.md#links-to-apis

Comment (yashints) [reply docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md]: Thank

**Review by mairaw — APPROVED** [review]
Great. Thank you @yashints. I've realized I forgot to add something, so I just changed that for you. As soon as the build finishes, I'm merging this.

**Pull request merged**